connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoaderTest.java

```diff
@@ -118,7 +119,8 @@ public void testLoadingMixOfValidAndInvalidPlugins() throws Exception {
 
         for (String sourceJar : TestPlugins.pluginPath()) {
             Path source = new File(sourceJar).toPath();
-            Files.copy(source, pluginPath.resolve(source.getFileName()));
+            Files.copy(source, pluginPath.resolve(source.getFileName()),
+                    StandardCopyOption.REPLACE_EXISTING);
```

Comment:
OOC, why is this added? Shouldn't the temporary `pluginDir` folder be cleaned up after test runs?

Reply:
Removing it doesn't seem to cause the test to fail, so I don't think it's necessary for the test to pass.

I believe you're correct that the pluginDir is cleaned up and won't have stale values from previous executions. A name collision may happen in other situations, none of which appear to be happening in this test:
* A test plugin produces an `invalid.jar` file that collides with the empty file created earlier in the test
* Two or more test plugins produce jar files with the same name

This may also have been necessary at some stage of development and then obviated earlier in the review.

I'll remove this change for clarity of the diff.